tutorials/mongoose/enums.md

```diff
@@ -0,0 +1,39 @@
+Mongoose `String` and `Number` types had access to the `enum` validator.
```

Comment:
"types have an `enum` validator`

Reply:
I noticed in fundamentals that you have an enum.md article. Do you want to link to it at all or?